runtime/scheduler.hpp

```diff
@@ -133,7 +134,8 @@ class Scheduler
      *
      * \return              Pointer to the target task
      */
-    Scheduler_task* get_task_by_id(uint16_t task_id) const;
+    const Scheduler_task* get_task_by_id(uint16_t task_id) const;
+    Scheduler_task* get_task_by_id(uint16_t task_id);
```

Comment:
Some modules have const access to scheduler and they can have const access to one task with the first method. Other modules have non-const access to Scheduler and they can have non-const access to a task.